src/main/java/de/tum/in/www1/artemis/service/connectors/bitbucket/BitbucketService.java

```diff
@@ -599,9 +583,10 @@ private void grantGroupPermissionToProject(String projectKey, String groupName,
      */
     private Map<Integer, String> getExistingWebHooks(String projectKey, String repositorySlug) throws BitbucketException {
```

Comment:
we should return `List<BitbucketWebHookDTO>` here and simply return `response.getBody().getSearchResults()` or `List.of()`

Reply:
good idea. I've fixed that